src/components/EarthOrbitsContainer.tsx

```diff
@@ -1,102 +1,133 @@
-            orbitIds
-                .filter((id) => id !== activeId)
-                .map((id) => {
-                    orbitElements.push(<SingleOrbitImage key={id + "_dotted"} id={id} imageDesc="_dotted" />);
-                })
-
-            //if there is an active tab AND a DIFFERENT tab is hovered
-            if (hoveringId && hoveringId !== activeId) {
-                orbitElements.push(<SingleOrbitImage key={hoveringId + "_solid"} id={hoveringId} imageDesc="_solid" />)
-            }
-        } else {
-            if (hoveringId) {
-                //place all orbit animations at a lighter opacity and not moving, unless it is the one that is hovered.  Note that this only occurs when no orbit is active/selected
-                orbitIds.map((id) => {
-                    if (hoveringId !== "gto") {
-                        orbitElements.push(<SingleOrbitAnimation
-                            key={id + "_orbit_moving"}
-                            id={id}
-                            player={Player}
-                            desc="_orbit_moving"
-                            isSelectedOrbit={id === hoveringId}
-                        />)
-
-                    } else {
-                        return (
-                            orbitElements.push(<SingleOrbitImage id={"gto"} imageDesc="_solid_lighter" />)
-                        );
-                    }
-                })
-            } else {
-                //no tab is hovered OR active/selected
-                orbitIds.map((id) => {
-                    if (id !== "gto") {
-                        orbitElements.push(<SingleOrbitAnimation
-                            key={id + "_orbit_moving"}
-                            id={id}
-                            player={Player}
-                            desc="_orbit_moving"
-                            isSelectedOrbit={true}
-                        />)
-                    } else {
-                        orbitElements.push(<SingleOrbitImage id={"gto"} imageDesc="_solid_active" />)
-                    }
-                })
-            }
-
-        }
-
-        return (
-            <div key="earth_orbits_container" id="earth_orbits_container" className="earth_orbits_container" >
-
-                <img src={earth} className="earth" alt="earth" />
-
-                {orbitElements}
-
-            </div>
+  const orbitIds = ["leo", "meo", "heo", "gso", "geo", "gto"];
+  const activeId = useSelector((state: RootState) => state.app.activeId);
+  const hoveringId = useSelector((state: RootState) => state.app.hoveringId);
+
+  const handleOrbitActivation = () => {
+    const orbitElements: JSX.Element[] = [];
+
+    //an orbit has been selected (active state)
+    if (activeId) {
+      if (activeId !== "gto") {
+        //the selected orbit will have animation
+        orbitElements.push(
+          <SingleOrbitAnimation
+            key={activeId + "_orbit_moving"}
```

Comment:
It lists all this stuff as new and a bunch of other stuff  as deleted but like I mention in a later comment I think it is formatting.

Please look it over and double check